### DISCUSSION TITLE
Bounty board has no prefered tool to mine it with

### DISCUSSION THREAD

**Discussion by cuberyl-catastrophe** [ejektaflex/Bountiful#283] [Mod Ideas]
### What happened?

![image](https://github.com/ejektaflex/Bountiful/assets/150861277/ff8f2644-b43a-47b3-a890-25ea8420f455)
Bounty boards have no prefered tool for mining them. This makes them take an annoying long time to mine. They should probably be minable with an axe/.

### Mod Version

6.0.3

### Mod Loader

Forge

### Mod Loader Version

47.2.20

**Comment by ejektaflex** [comment]
This is true! I originally did not have a preferred tool because I didn't want the board to accidentally get mined, but if we tweak the block hardness we can make it feel good. I'll put this on my todo list for the next version. Thanks!

**Comment by vizthex123** [comment]
came here to report the same thing lol

tried adding the `mineable/axe` tag, but it didn't work :'c

**Comment by ejektaflex** [comment]
I'm going to convert this to a discussion, since it is not a bug but rather it was originally purposeful. I still plan to change this in a future version.

**Comment by vizthex123** [comment]
I did get it working by just adding the `minecraft:mineable/axe` tag. Perhaps it could be a config option?

That or giving it a decent hardness (above logs but not like wooden obsidian or anything lol) would be fine - especially if it retained its contents, that way you don't lose anything if you break it.

**Comment by TheDeviantCrafter** [comment]
Maybe copy Lootr?

Dungeon chests with Lootr installed can be looted by multiple players. Trying to break one without holding sneak fails, and triggers a chat message suggesting that the player leave it for others. Holding sneak suppresses this warning and allows the player to break it anyway.
